metadata/src/lyrics.rs

```diff
@@ -0,0 +1,79 @@
+use bytes::Bytes;
+
+use librespot_core::{Error, FileId, Session, SpotifyId};
+
+impl LyricsReply {
```

Comment:
Maybe just `Lyrics`?

Reply:
Agreed, but then I need to change the name of the other inner struct. `LyricsInner` ? `LyricsDetails`?

Reply:
Inner sounds good to me.